gcp/pd-backed-nfs-server/manifests/pv-for-nfs-server.yaml

```diff
@@ -1,31 +0,0 @@
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: env0-pv-for-nfs-server
-spec:
-  storageClassName: env0-sc-for-nfs-server
-  capacity:
-    storage: 300Gi
-  accessModes:
-  - ReadWriteOnce
```

Comment:
if we remove those files, don't we lose the full-blown installation for GCP?
maybe we should create another **_optional_** PV module that will connect to the new helm chart?

Reply:
Since this is just a reference example for now I think that should suffice.  
We should add more control when we make it a public configurable Terraform module (similar to the effort @eladmosh did for AWS)